### DISCUSSION TITLE
Can I alter the default directory structure → link config?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [vuejs/vitepress#1133] [Q&A] [answered]
I'd like to be able to organize my files using folders while still being able to automate bits of the URL pattern.

i.e. `/docs/articles/vue/heres-an-article.md` is rendered at `site.com/heres-an-article.html` rather than the default of `site.com/articles/vue/heres-an-article.html`

Is this currently configurable?

**Comment by brc-dd** [comment, marked as answer]
> Is this currently configurable?

No. You can track #285, but it's not on priority RN.

**Reply by CodyBontecou** [reply]
Good to know, thank you.
